bokeh/embed.py

```diff
@@ -30,23 +30,25 @@
 from .util.serialization import make_id
 from .util.deprecation import deprecated
 
-def _prefix(text, prefix):
-    return "\n".join([ prefix + line for line in text.split("\n") ])
-
-def _indent(text):
-    return _prefix(text, "    ")
-
-def _wrap(pre, text, post):
-    return '%s%s%s' % (pre, _indent(text), post)
-
-def _wrap_in_function(code):
-    return _wrap('Bokeh.$(function() {\n', code, '\n});')
+def _indent(text, n=2):
+    return "\n".join([ " "*n + line for line in text.split("\n") ])
 
 def _wrap_in_safely(code):
-    return _wrap('Bokeh.safely(function() {\n', code, '\n});')
+    return """\
+Bokeh.safely(function() {
+%(code)s
+});""" % dict(code=_indent(code, 2))
 
 def _wrap_in_onload(code):
-    return _wrap('document.addEventListener("DOMContentLoaded", function(event) {\n', code, '\n});')
+    return """\
+(function() {
+  var fn = function() {
+%(code)s
+  };
+  if (document.readyState != "loading") fn();
+  else document.addEventListener("DOMContentLoaded", fn);
+})();
+""" % dict(code=_indent(code, 4))
```

Comment:
Indentation in bokehjs consists of two spaces. Here it was all four and in other files mixed 2 and 4.